packages/components/dropdown/src/dropdown.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div :class="ns.b()">
+  <div :class="[ns.b(), { disabled }]">
```

Comment:
please use `is-disabled` instead of disabled

Reply:
Resolved

ps: I just search the whole project, and found `disabled` classname in other files. I think they should use `is-disabled` too. If it does, i'll start another pr to do it

Reply:
@freedomlang Sure! Let's refactor them!

Reply:
@freedomlang seems like you didn't do `ns.is('disabled', disabled)` as @msidolphin asked. Would you mind update it? I am preparing a release now

Reply:
> @freedomlang seems like you didn't do `ns.is('disabled', disabled)` as @msidolphin asked. Would you mind update it? I am preparing a release now

I'll check it in a secsond

Reply:
> > @freedomlang seems like you didn't do `ns.is('disabled', disabled)` as @msidolphin asked. Would you mind update it? I am preparing a release now
> 
> I'll check it in a secsond

Done